resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,11 @@
 
   <change-notes>
     <![CDATA[
+0.1.8.1:
+<ul>
+  <li>improve handling of breakpoints for the bazel launch config</li>
```

Comment:
For completeness we might add a line item to record that the iOS simulator now gets activated/brought to front on run/debug.